dockers/docker-snmp/Dockerfile.j2

```diff
@@ -44,7 +44,9 @@ RUN locale-gen
 RUN pip3 install --no-cache-dir \
         hiredis                             \
         pyyaml                              \
-        smbus
+        smbus                               \
+        aioredis==1.3.0                     \
```

Comment:
Why not install the latest version? Is there anything disruptively changed after 1.3.0?

Reply:
We tried with version 2.0.1 and trap generation is not working. We need to debug further to find why its failing.
Version  1.3.0 is working fine.